src/SelectSingle.vue

```diff
@@ -105,16 +108,32 @@
 				return this.disabled || this.loading
 			},
 			selectedIndex(): number {
-				return this.value
-				? this.options.findIndex(currentOption => currentOption.value == this.value.value)
```

Comment:
This was the actual bug in SelectSingle re: which option was showing as selected in the dropdown, when using `uniqueIdField`